src/web/jcc/client/components/TaskSettingsPanel.tsx

```diff
@@ -85,6 +85,20 @@ function TaskOptionsPanel(props: Props) {
         props.onChange(newTask)
     }
 
+    function onChangeStationKeepParameter(evt: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
Hey Nick, can you add a function comment here and switch the `var` to `let`. I know all the other related functions use `var` but lets stop that trend.